web/thesauruses/python/3/queues_stacks.json

```diff
@@ -0,0 +1,388 @@
+        ]
+      },
+      "queue_duplicate_next_item": {
+        "name": "Duplicate next item",
+        "code": [
+          "stack.append(stack[-1])"
+        ]
+      },
+      "queue_swap_two_items": {
+        "name": "Swap two items",
+        "code": [
+          "mylist = ['One','Two','Three', 'Four']\npos1, pos2  = 0, 2\nmylist[pos1], mylist[pos2] = mylist[pos2], mylist[pos1]"
+        ]
+      },
+      "queue_get_size": {
+        "name": "Get size (number of items) in the queue",
+        "code": [
+          "print(len(stack))"
+        ]
+      },
+      "queue_get_capacity": {
+        "name": "Check capacity of queue",
+        "not-implemented": "true"
+      },
+      "queue_export_to_list": {
+        "name": "Export a list of all queue items",
+        "code": [
+          "mylist = queue"
+        ]
+      },
+      "queue_clear_all": {
+        "name": "Clear out all queue items",
+        "code": [
+          "queue.clear()"
+        ]
+      },
+      "stack_push_item": {
+        "name": "Push an item (add to top)",
+        "code": [
+          "stack.append('Four')"
+        ]
+      },
+      "stack_pop_return_item": {
+        "name": "Pop an item (remove from top, return item)",
+        "code": [
+          "print(stack.pop())"
+        ]
+      },
+      "stack_pop_delete_item": {
+        "name": "Pop an item (remove from top, don't return)",
+        "code": [
+          "stack.pop()"
+        ]
+      },
+      "stack_peek_at_next_item": {
+        "name": "Look/peek at next element (from top)",
+        "code": [
+          "print(stack[-1])"
+        ]
+      },
+      "stack_peek_at_last_item": {
+        "name": "Look/peek at last element (from bottom)",
+        "code": [
+          "print(stack[0])"
+        ]
+      },
+      "stack_duplicate_next_item": {
+        "name": "Duplicate top item",
+        "code": [
+          "stack.append(stack[-1])"
+        ]
+      },
+      "stack_swap_two_items": {
+        "name": "Swap two items",
+        "code": [
+          "stack = ['One','Two','Three', 'Four']\npos1, pos2  = 0, 2\nstack[pos1], stack[pos2] = stack[pos2], stack[pos1]"
+        ]
+      },
+      "stack_get_size": {
+        "name": "Get size (number of items) on the stack",
+        "code": [
+          "print(len(stack))"
+        ]
+      },
+      "stack_get_capacity": {
+        "name": "Check capacity of stack",
+        "not-implemented": "true"
+      },
+      "stack_export_to_list": {
+        "name": "Export a list of all stack items",
+        "code": [
+          "mylist = stack"
+        ]
+      },
+      "stack_clear_all": {
+        "name": "Clear all queue items",
+        "code": [
+          "stack.clear("
+        ]
+      },
+      "iterate_pointer_data_type": {
+        "name": "Data type of a iterator pointer",
+        "not-implemented": "true"
+      },
+      "iterate_create_pointer": {
+        "name": "Create iterator",
+        "code": [
+          "pointer = 0"
```

Comment:
If iterators/pointers are "not-implemented", then these should be "not-implemented" too. Otherwise they're integers and not iterators.